docs/iris/src/developers_guide/code_format.rst

```diff
@@ -0,0 +1,33 @@
+.. _iris_code_format:
+
+Code Formatting
+***************
+
+Iris uses `black <https://black.readthedocs.io/en/stable/>`_ formatting and `flake8 <https://flake8.pycqa.org/en/stable/>`_
+linting to enforce a consistent code format throughout the project. ``black`` and ``flake8`` can easily be installed
+into your development environment with ``pip``::
+
+  $ pip install black flake8
+
+and then manually run from the root of the Iris repository::
+
+  $ black .
+  $ flake8 .
+
+Alternatively, we recommend using `pre-commit <https://pre-commit.com/>`_ to automatically run ``black`` and ``flake8``
```

Comment:
It would be worth bringing up that there is good information on IDE integration in the black documentation (and probably also the flake8 documentation, though I haven't looked so much at that myself).